website/docs/r/efs_access_point.html.markdown

```diff
@@ -0,0 +1,73 @@
+---
+subcategory: "EFS"
+layout: "aws"
+page_title: "AWS: aws_efs_access_point"
+description: |-
+  Provides an Elastic File System (EFS) access point.
+---
+
+# Resource: aws_efs_access_point
+
+Provides an Elastic File System (EFS) access point.
+
+## Example Usage
+
+```hcl
+resource "aws_efs_access_point" "test" {
+  file_system_id = "${aws_efs_file_system.foo.id}"
+  subnet_id      = "${aws_subnet.alpha.id}"
```

Comment:
Sorry I missed this the first review -- this resource does not support this argument.

```suggestion
```

Reply:
dont copy paste and drink